tracing/src/lib.rs

```diff
@@ -214,7 +214,7 @@
 //! ### Configuring Attributes
 //!
 //! Both macros require a [`Level`] specifying the verbosity of the span or
-//! event. Optionally, the [target] and [parent span] may be overridden. If the
+//! event. Optionally, the name, [target] and [parent span] may be overridden. If the
```

Comment:
I don't think we should state "optionally the name can be overridden" here, since it's not actually an optional override for the span macros, and this paragraph is discussing attributes that are overridable for both macros. I think it's best to just discuss it in the second paragraph you've added around names.
```suggestion
//! event. Optionally, the, [target] and [parent span] may be overridden. If the
```